ui/plugins/pluginpanel.html

```diff
@@ -206,6 +206,13 @@
           <label>Log parsing status:</label>
           <span>No logs</span>
         </li>
+        <li class="small" ng-click="backfillJob()">
+          <label>
+            <a title="Trigger this jobtype back to the previous time it was run">
+                Backfill this job
+            </a>
+          </label>
+        </li>
```

Comment:
Thanks for that bug. Right now since this is a sheriff-only action, I guess sheriffs are ok to have it in the job details pane, we can move the "backfill" to action button when bug 1113264 is fixed.